src/Socket/chats.ts

```diff
@@ -190,18 +190,20 @@ export const makeChatsSocket = (config: SocketConfig) => {
 		}).filter(item => item.exists)
 	}
 
-	const fetchStatus = async(jid: string) => {
-		const [result] = await interactiveQuery(
-			[{ tag: 'user', attrs: { jid } }],
+	const fetchStatus = async(...jids: string[]) => {
+		const list = jids.map((jid) => ({ tag: 'user', attrs: { jid } }))
+		const results = await interactiveQuery(
+			list,
 			{ tag: 'status', attrs: {} }
 		)
-		if(result) {
-			const status = getBinaryNodeChild(result, 'status')
+		return results.map(item => {
+			const status = getBinaryNodeChild(item, 'status')
 			return {
+				user: item.attrs.jid,
 				status: status?.content!.toString(),
```

Comment:
```ts
status: status && status.content ? status.content.toString() : null,
```

prevent undefined error if contact doesn't have status

additional info in readme need to change to

```ts
const jid = ["abx@s.whatsapp.net","xyz@s.whatsapp.net]
const status = await sock.fetchStatus(...jid)
console.log("status: " + status)
```

Not sure if we need this, but don't have any problem if others agree

Reply:
i didn't change the original code in this part, I don't think that's necessary

Reply:
> i didn't change the original code in this part, I don't think that's necessary, the operator [ ! ] already do this

the current code (old) indeed has a problem if the status is undefined. It is better for us to handle this issue in this pr

status: status?.content!.toString() will throw an error

```txt
TypeError: Cannot read properties of undefined (reading 'toString')
    at /home/node_modules/@whiskeysockets/baileys/lib/Socket/chats.js:170:88
    at Array.map (<anonymous>)
    at Object.fetchStatus (/home/node_modules/@whiskeysockets/baileys/lib/Socket/chats.js:166:24)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    ```
    
My suggestion is to safely handles cases where status or status.content might be undefined or null. we should returns null for the status property if status or status.content is not available..

Reply:
applied